src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -422,9 +422,7 @@ public static boolean isTextPresentInsideHtmlTag(DetailNode node) {
      * @return true, if the inline tag node is present.
      */
     private static boolean isInlineTagPresent(DetailNode ast) {
-        return ast.getType() == JavadocTokenTypes.JAVADOC_INLINE_TAG
-                || ast.getType() == JavadocTokenTypes.HTML_ELEMENT
-                && getInlineTagNodeWithinHtmlElement(ast) != null;
+        return getInlineTagNodeWithinHtmlElement(ast) != null;
```

Comment:
You need to rename `getInlineTagNode(ast)`

WithinHtmlElement is confusing now.
And maybe we need to inline content of isInlineTagPresent to avoid extra method.

Reply:
Done renamed